test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests/Get_ApplicationJson_DictionaryObject_Tests.cs

```diff
@@ -71,6 +71,9 @@ public void Given_OpenApiDocument_Then_It_Should_Return_ComponentSchema(string @
         [DataTestMethod]
         [DataRow("dictionaryObjectModel", "int32ObjectValue", "object", "int32ObjectModel")]
         [DataRow("dictionaryObjectModel", "stringObjectModel", "object", "stringObjectModel")]
+        [DataRow("dictionaryObjectModel", "stringArrayValue", "object", "string[]_string")]
```

Comment:
In the OpenAPI spec, the Dictionary type is always Dictionary<string, ....>. It can't be Dictionary<object, ...>.